.github/workflows/test_cocoapods_integration.yml

```diff
@@ -6,7 +6,7 @@ on:
 
   push:
     branches:
-    - master
+    - main
```

Comment:
technically, we can remove mentioning of both `main` and `master` here - this branch is obsolete and only exist since it contains tags for versions v1-v3